src/constants/defaults.ts

```diff
@@ -0,0 +1,5 @@
+import { ETHEREUM_NULL_ADDRESS } from '../types/ethereum'
+
+export const defaultAdminAddress = () => {
+  return process.env.DEFAULT_ADMIN_ADDRESS?.toLowerCase() || ETHEREUM_NULL_ADDRESS
+}
```

Comment:
don't need to use a function here, can just use the value directly?

also, should maybe just throw error & crash if there is no DEFAULT_ADMIN_ADDRESS set?